### PR TITLE
added config file for SAST tools Checkov

### PR DESCRIPTION
**Pull request by xinredhat**
**Changes:**

Added config file for SAST tools Checkov. The Checkov [CI PR](https://github.com/openshift/release/pull/31716) relies on this PR.

You could reference [CLI Command Reference](https://www.checkov.io/2.Basics/CLI%20Command%20Reference.html) for the the properties in config file.

**Verify:**
You can execute the command with its Docker image to see the result.
```
$ docker run -it --rm --volume <the path of pipelines-service directory>:/tf --workdir /tf bridgecrew/checkov  --directory /tf --config-file /tf/test/config/checkov.yaml
```

### CODE REVIEW

**Review by Roming22 — APPROVED** [review]

File: test/config/checkov.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+compact: false
+evaluate-variables: true
+external-checks-dir:
+  - docs
+  - exploration/argocd-external/docs
+framework:
+  - kubernetes
+  - dockerfile
+  - github_actions
+  - gitlab_ci
```
Comment (fgiloux) [inline test/config/checkov.yaml:12]: Why do we need gitlab_ci?

File: test/config/checkov.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - kustomize
+  - argo_workflows
```
Comment (fgiloux) [inline test/config/checkov.yaml:14]: I don't think we use argo workflows, do we?

Comment (xinredhat) [reply test/config/checkov.yaml]: you are right.  We don't have the below  argo workflows.  Fixed.

```
apiVersion: argoproj.io/v1alpha1
kind: Workflow
```

File: test/config/checkov.yaml (hunk 1 above)
Comment (xinredhat) [reply test/config/checkov.yaml]: fixed.

**Review by bnallapeta — APPROVED** [review]
LGTM. @fgiloux if your earlier comments are addressed, please go ahead and merge the PR.

**Pull request merged**